armi/runLog.py

```diff
@@ -310,20 +310,30 @@ def concatenateLogs(logDir=None):
 
     info("Concatenating {0} log files".format(len(stdoutFiles)))
 
+    dirName = os.path.dirname(stdoutFiles[0])
```

Comment:
Can you add some sort of unit testing for this new code you added?

(There are already a ton of `runLog.py` tests in `armi/tests/test_runLog.py`, you could probably snag what you need there.)

Thanks!

Reply:
I added a little to `test_concatenateLogs` to improve the coverage.